src/etherswallet/account.js

```diff
@@ -70,6 +70,18 @@ module.exports.newAccount = (encryptedJson, decryptedWallet = null) => {
       const signedMsg = ethUtil.bufferToHex(concatSig(sig.v, sig.r, sig.s));
       cb(null, signedMsg);
     },
+    sign: ({ msg, chainId }, cb) => {
```

Comment:
[Question] DId you get any conclusions about what is the different between `signMsg` logic and this one? 

IMO I would be more include to call this method something less generic and confusing as `sign`, maybe `signRawMsg` makes more sense, por `rpcSignMsg`.

Reply:
I like to "signRawMsg", is more explanatory. The main difference is that this one doesn't do some of the hashing/encoding and also returns a different output that's why I can't reuse the orig method. Will update it.